Tools/TestWebKitAPI/Tests/WebKitCocoa/SiteIsolation.mm

```diff
@@ -1359,4 +1359,37 @@ HTTPServer server({
     } while (!mainFrameInfo.get()._isFocused || childFrameInfo.get()._isFocused);
 }
 
+TEST(SiteIsolation, EvaluateJavaScriptInFrame)
+{
+    HTTPServer server({
+        { "/mainframe"_s, { "<iframe id='iframe' src='https://domain2.com/subframe'></iframe>"_s } },
+        { "/subframe"_s, { "<script>test = 'abc';</script>"_s } }
+    }, HTTPServer::Protocol::HttpsProxy);
+
+    auto navigationDelegate = adoptNS([TestNavigationDelegate new]);
```

Comment:
It looks like you can use siteIsolatedViewAndDelegate here to avoid boilerplate code.